src/app/Instance/BridgeOverview/Story/BOEmptyState.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { BOEmptyState } from "@app/Instance/BridgeOverview/Components/BOEmptyState";
+
+export default {
+  title: "Bridge/Bridge overview/Bridge overview empty state",
```

Comment:
After adding a story for the main component, could you move _all_ the children components stories inside a subfolder?

Like `title: "Bridge/Bridge overview/Components/Bridge overview empty state",`